src/components/migrationToV3.stories.mdx

```diff
@@ -0,0 +1,119 @@
+<Meta title="Migration to V3" />
+
+# Migration from 2.0 to 3.0
+
+Version 3 of the Amazon Chime React Component Library makes small changes to the interface of the library to make it more customizable.
+
+### IMPORTANT - Version 3 of the library also requires a dependency to version 3.0.0 or higher of the amazon-chime-sdk-js.
+
+In many cases, you will need to adjust your application code because interface parameters to root providers like `MeetingProvider` are updated.
+
+## Installation
+
+Installation involves adjusting your package.json to depend on version 3.x.x
+
+```
+npm uninstall amazon-chime-sdk-component-library-react amazon-chime-sdk-js
+npm install --save amazon-chime-sdk-component-library-react@beta amazon-chime-sdk-js@beta
```

Comment:
@ltrung The npm install command will need to be updated after release of V3. During beta, we will have to use the `@beta` tags.